docs-src/stargate-core/modules/install/pages/building.adoc

```diff
@@ -1,7 +1,41 @@
 = Building Stargate from source
 :page-tag: stargate,dev,develop,install
 
-In order to build Stargate, you'll need to have https://adoptopenjdk.net/[Java 8 (jdk8)] installed.
+This page covers the basics of building Stargate v1 or v2 from source. If you want further details, see
+the Developer Guide READMES:
+
+* https://github.com/stargate/stargate/blob/v2.0.0/DEV_GUIDE.md[v2 Developer Guide]
```

Comment:
Note that referencing the v2 branch is fine for now but will need to be changed when v2 goes GA and we rename this branch to `main`.

Reply:
I'll will need to ask another writer to change this on Oct 26 and republish.